src/test/java/org/zalando/putittorest/KeystoreConfigurationTest.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.putittorest;
+
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.zalando.putittorest.annotation.RestClient;
+import org.zalando.riptide.Rest;
+
+import javax.net.ssl.SSLHandshakeException;
+
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.zalando.riptide.Bindings.anySeries;
+import static org.zalando.riptide.Navigators.series;
+import static org.zalando.riptide.Route.pass;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = DefaultTestConfiguration.class)
+public class KeystoreConfigurationTest {
```

Comment:
> To test this locally one would need to setup a local SSL secured HTTP server, right?

I looked, but haven't found any simple solution... any idea?

> I'd also suggest to use github.com, rather than yourturn.

:+1: